controller/logs.go

```diff
@@ -46,17 +45,18 @@ func (c *Controller) LogsForDeployment(ctx context.Context, req *entity.Deployme
 		return err
 	}
 	// Break them down line by line
-	logLines := strings.Split(deploy.DeployLogs, "\n")
+	logLines := strings.Split(fetchCurrentLogs(deploy), "\n")
 	offset := 0.0
 	if req.NumLines != 0 {
 		// If a limit is set, walk it back n steps (with a min of zero so no panics)
 		offset = math.Max(float64(len(logLines))-float64(req.NumLines)-1, 0.0)
 	}
 	// Output Initial Logs
-	fmt.Print(strings.Join(logLines[int(offset):], "\n"))
+	fmt.Println(strings.Join(logLines[int(offset):], "\n"))
 	if req.NumLines == 0 {
 		// If no log limit is set, we stream logs
-		prevLogs := strings.Split(deploy.DeployLogs, "\n")
+		idxMp := make(map[string][]string)
+		idxMp[deploy.Status] = strings.Split(deploy.DeployLogs, "\n")
```

Comment:
Is this actually correct? Shouldn't it be giving build or deploy logs depending on what the status is?